README.md

```diff
@@ -15,21 +15,24 @@ The pipeline is built using Nextflow, a workflow tool to run tasks across multip
 
 1. Raw read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
 2. Adapter trimming ([`Trim Galore!`](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/))
-3. Alignment against miRBase mature miRNA ([`Bowtie1`](http://bowtie-bio.sourceforge.net/index.shtml))
-    1. Post-alignment processing of miRBase mature miRNA counts ([`SAMtools`](https://sourceforge.net/projects/samtools/files/samtools/))
-    2. Analysis on miRBase mature miRNA counts ([`edgeR`](https://bioconductor.org/packages/release/bioc/html/edgeR.html))
-         * TMM normalization and a table of top expression mature miRNA
-         * MDS plot clustering samples
-         * Heatmap of sample similarities
-4. Alignment against miRBase hairpin for the unaligned reads in step 3 ([`Bowtie1`](http://bowtie-bio.sourceforge.net/index.shtml))
-    1. Post-alignment processing of miRBase hairpin counts ([`SAMtools`](https://sourceforge.net/projects/samtools/files/samtools/))
-    2. Analysis on miRBase hairpin counts  ([`edgeR`](https://bioconductor.org/packages/release/bioc/html/edgeR.html))
-         * TMM normalization and a table of top expression hairpin
-         * MDS plot clustering samples
-         * Heatmap of sample similarities
-5. Alignment against host reference genome ([`Bowtie1`](http://bowtie-bio.sourceforge.net/index.shtml))
+3. Basic annotation
+    3.1 Alignment against miRBase mature miRNA ([`Bowtie1`](http://bowtie-bio.sourceforge.net/index.shtml))
+        1. Post-alignment processing of miRBase mature miRNA counts ([`SAMtools`](https://sourceforge.net/projects/samtools/files/samtools/))
```

Comment:
This table still looks a bit odd compared to:
https://github.com/nf-core/smrnaseq/tree/dev#pipeline-summary

Reply:
sorry, I don't know what to do with that. I cannot get the same format, and I removed some information because it was too much I think. Happy to try something if somebody gives me some tips.

Reply:
1. Raw read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
2. Adapter trimming ([`Trim Galore!`](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/))
3. Alignment against miRBase mature miRNA ([`Bowtie1`](http://bowtie-bio.sourceforge.net/index.shtml))
    1. Post-alignment processing of miRBase mature miRNA counts ([`SAMtools`](https://sourceforge.net/projects/samtools/files/samtools/))
    2. Analysis on miRBase mature miRNA counts ([`edgeR`](https://bioconductor.org/packages/release/bioc/html/edgeR.html))
         * TMM normalization and a table of top expression mature miRNA
         * MDS plot clustering samples
         * Heatmap of sample similarities
4. Alignment against miRBase hairpin for the unaligned reads in step 3 ([`Bowtie1`](http://bowtie-bio.sourceforge.net/index.shtml))
    1. Post-alignment processing of miRBase hairpin counts ([`SAMtools`](https://sourceforge.net/projects/samtools/files/samtools/))
    2. Analysis on miRBase hairpin counts  ([`edgeR`](https://bioconductor.org/packages/release/bioc/html/edgeR.html))
         * TMM normalization and a table of top expression hairpin
         * MDS plot clustering samples
         * Heatmap of sample similarities
5. Alignment against host reference genome ([`Bowtie1`](http://bowtie-bio.sourceforge.net/index.shtml))
    1. Post-alignment processing of alignment against host reference genome ([`SAMtools`](https://sourceforge.net/projects/samtools/files/samtools/))
6. Collapse reads ([`seqcsluter`](https://seqcluster.readthedocs.io/mirna_annotation.html#processing-of-reads))
7. Alignment agains miRBase hairpin ([`Bowtie1`](http://bowtie-bio.sourceforge.net/index.shtml))
8. miRNA and isomiR annotation using the previous alignment ([`mirtop`](https://github.com/miRTop/mirtop))
9. miRNA quality control ([`mirtrace`](https://github.com/friedlanderlab/mirtrace))
10. Present QC for raw read, alignment, and expression results ([`MultiQC`](http://multiqc.info/))

Reply:
Ive tried to merge your changes with the ones in https://github.com/nf-core/smrnaseq/tree/dev#pipeline-summary. Does that describe the pipeline appropriately? Maybe some issues with tabbing in the markdown formatting you originally had.

Reply:
Just hit `edit` on the comment to get the raw markdown :+1:

Reply:
thanks. I committed the changes, I worked directly on GitHub, I think my rendering wasn't working like this one. Let me know. (I changed some steps to match what happens in the code now.)

Reply:
Ok. Now Im confused  because the `README.md` doesnt have the same bullets I sent in the comment above. It looks like the old `README.md`. Is that intentional?